### ISSUE TITLE
NRLv2 further work

### ISSUE THREAD

**Issue by megies** [obspy/obspy#3122]
This is a follow up ticket for NRLv2, continuing after #3058 (more details in that old ticket)

check list:
 - [x] revert a commit fixing duplicated sections in downloaded NRL v2
 - [x] statement from IRIS/EarthScope is that the navigational txt files that were included in NRLv1 will not be implemented/served in the NRLv2, even though they are present in the downloaded version, so there will be no online client support for NRL anymore once they finally pull the plug on NRLv1 which was said to happen spring 2023 ~~read NRL v2 online? at the moment not really possible, new online version does not serve the txt files with the traversal through the structure. In principle possible to parse the html files, but.. let's not~~
 - [x] add tests (for offline version)? current tests only use NRL v1 online
 - [x] NRL v2 has additional base nodes "integrated" and "soh" (in addition to "sensor" and "datalogger"), maybe check if we want to implement that too, but might be for another PR
 - [x] fix some issues with units in the first datalogger stage when combining responses using the NRL v2
 - [ ] ...
 - [x] add changelog
